frontend/package.json

```diff
@@ -14,6 +14,7 @@
     "@shoutem/ui": "^0.23.12",
```

Comment:
(commenting on the line closest to the desired line)

Right now your test script (which is used by the CI) is `jest --updateSnapshot`.
Furthermore, I don't see any snapshots that are actually committed to Git.

Effectively this means that your "tests" are generating the snapshots and always declaring them as correct, but since they have nothing to compare against nothing is being tested at all on the front end right now.

This command should just be `jest`, and snapshots should be committed to Git alongside the tests, with `jest --updateSnapshot` only being used locally when a component is changed or added (after verifying that the output is indeed correct), signaling that something changed.

Reference: [https://jestjs.io/docs/en/snapshot-testing](https://jestjs.io/docs/en/snapshot-testing)

Reply:
Yes you are correct, this is being addressed in another Pull Request